### PR TITLE
[BEAM-1270] Fix display data for ReadFromText and WriteToText

### PR DESCRIPTION
**Pull request by pabloem**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by katsiapis** [comment]
Can we also update the Beam tfrecordio file(s) appropriately regarding display_data (if it makes sense)?
And similarly for other IOs (like avroio etc).

**Comment by pabloem** [comment]
@katsiapis because the `tfrecordio.py` transforms and sinks already inherit and use the basic `Write`, `Read`, `FileBasedSource`, and `FileBasedSink` (which is what I've fixed), the display data should come to the TF transforms/sinks/sources.
r: @chamikaramj can you take a look? : )

**Comment by pabloem** [comment]
r: @aaltay since Cham is on vacation

File: sdks/python/apache_beam/io/textio.py [hunk 1]
```diff
@@ -83,6 +83,13 @@ def __init__(self, file_pattern, min_bundle_size,
     self._coder = coder
     self._buffer_size = buffer_size
 
+  def display_data(self):
```
Comment (aaltay) [inline sdks/python/apache_beam/io/textio.py]: How about the other parameters (buffer_size)?

Comment (pabloem) [inline sdks/python/apache_beam/io/textio.py]: Do we also want display data for the coder?

Comment (aaltay) [inline sdks/python/apache_beam/io/textio.py]: What do we do in other places? Do we normally include everything or try to come up with a selected set? If the answer is not everything, I think we do not need to display neither buffer_size nor coder.

Comment (pabloem) [inline sdks/python/apache_beam/io/textio.py]: We normally give as much as possible. I'll add the coder since it's something users may want to find out.

Comment (pabloem) [inline sdks/python/apache_beam/io/textio.py]: Done.

**Comment by pabloem** [comment]
r: @chamikaramj Since you're back, can you take a look at this? : )

**Comment by pabloem** [comment]
@chamikaramj can you take a look please? It's a very small change.

**Comment by chamikaramj** [comment]
LGTM

**Comment by pabloem** [comment]
The test failure is unrelated. It has to do with dependencies, and it reproduces also in [PR 1845](https://builds.apache.org/job/beam_PreCommit_Java_MavenInstall/6812/).
@robertwb can you merge please?

**Pull request closed without merging**